mod-source-record-manager-server/src/main/java/org/folio/verticle/consumers/consumerstorage/KafkaConsumersStorageImpl.java

```diff
@@ -0,0 +1,39 @@
+package org.folio.verticle.consumers.consumerstorage;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.folio.kafka.KafkaConsumerWrapper;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+@Component
+public class KafkaConsumersStorageImpl implements KafkaConsumersStorage {
+  private static final Logger LOGGER = LogManager.getLogger();
+
+  private final Map<String, List<KafkaConsumerWrapper<String, String>>> consumerWrappersMap = Collections.synchronizedMap(new HashMap<>());
```

Comment:
nitpick: I have not used sychronizedMap before, typically I use [ConcurrentHashMap](https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/ConcurrentHashMap.html) if i need synchronization with a hashmap.

According to this [article](https://www.baeldung.com/java-synchronizedmap-vs-concurrenthashmap) ConcurrentHashMap doesn't allow nulls and supposed to be faster than synchronizedMap. I am fine if this is left as is.

Reply:
Im ok with using ConcurrentHashMap, made this change